packages/accounts-controller/src/AccountsController.ts

```diff
@@ -1131,7 +1131,7 @@ export class AccountsController extends BaseController<
    *
    * @param id - The EVM client ID or non-EVM chain ID that changed.
    */
-  readonly #handleMultichainNetworkChange = (
+  readonly #handleOnMultichainNetworkChange = (
```

Comment:
Almost 🙈 the name is still incorrect, it should be:

```suggestion
  readonly #handleOnMultichainNetworkDidChange = (
```